### PR TITLE
Website redesign

### PR DESCRIPTION
**Pull request by HaSistrunk**
Redesign of projectelectron.rockarch.org with new multi-page Jekyll theme. Compiles project content including transfer specification, project values, user research artifacts, and basic information about the project.

The `components.md` page needs attention, as I would like a review of how much detail to include (particularly in the Repository and Discovery sections at the end).

In general, a fresh eye to find any glaring problems or omissions would be appreciated, and any feedback on design is welcome.

### CODE REVIEW

**Comment by HaSistrunk** [comment]
I made some changes - implemented cards, embedded the gist scripts into the site, and made a few styling tweaks. @helrond let me know if you think this needs anything else before merging.

**Review by helrond — APPROVED** [review]
I made a few tweaks to the cards to compact the display a bit more. Ready to merge if you're okay with the changes!

**Pull request merged**
